### PR TITLE
ASM-7640 Warning Messages thrown in FC 630 cluster for VSAN

### PR DESCRIPTION
**Pull request by sushilrai**
Addressing the code comments which were added to the merged code. Added exception handling block for teardown.

### CODE REVIEW

File: lib/puppet/provider/vc_vsan_health_performance/default.rb [hunk 1]
```diff
@@ -25,7 +28,7 @@ def exists?
     if resource[:ensure] == :present
       group_health != "green" ? false : true
     elsif resource[:ensure] == :absent
-      group_health != "green" ? false : true
+      group_health != "unknown" ? true: false
```
Comment (ripienaar) [inline lib/puppet/provider/vc_vsan_health_performance/default.rb]: this is fine if you find it more readable but its equiv to just `group_health != "unknown"` no need for the `? true : false`

**Comment by ripienaar** [comment]
looks good to me, I didnt review the earlier issues though so I'll defer to who reviewed the other

**Comment by sushilrai** [comment]
@kaushalgala please review the PR

File: lib/puppet/provider/vc_vsan_health_performance/default.rb [hunk 2]
```diff
@@ -1,11 +1,10 @@
 provider_path = Pathname.new(__FILE__).parent.parent
 require File.join(provider_path, 'vcenter')
-require 'rbvmomi'
 require File.join(provider_path, 'vsanmgmt.api')
 require File.join(provider_path, 'vsanapiutils')
 
 Puppet::Type.type(:vc_vsan_health_performance).provide(:vc_vsan_health_performance, :parent => Puppet::Provider::Vcenter) do
-  @doc = "Enable / Disable VSAN property."
+  @doc = "Enable / Disable VSAN Health Performannce Service."
```
Comment (kaushalgala) [inline lib/puppet/provider/vc_vsan_health_performance/default.rb]: Nit: spelling for Performannce -> Performance :)

**Comment by kaushalgala** [comment]
Other than a minor spelling mentioned above, everything else looks good :+1:

**Comment by kaushalgala** [comment]
@ripienaar can you merge this PR, looks fine

**Pull request merged**